usr/bin/hwctl

```diff
@@ -91,7 +94,7 @@ function system-info {
 	\"autotestVersion\": \"$AUTOTEST_VERSION\",
 	\"playserveVersion\": \"$PLAYSERVE_VERSION\",
 	\"gridVersion\": \"$GRID_VERSION\",
-	\"playtronLabsVersion\": \"$PLAYTRON_LABS_VERSION\",
```

Comment:
We are using `playtronLabsVersion` in Grid. There needs to be a sibling PR to update this variable in Grid as well.

Reply:
PR to Grid made

Reply:
@strycore Did that PR get merged to Grid? I can't seem to find it. Do we need any changes to the backend code or can we merge this now?

Reply:
We got confirmation that no backend changes are needed.